lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -212,9 +212,11 @@ def post(self, request):
         track.views.server_track(request, action, {}, page='user_sysdashboard')
 
         if action == 'download_users':
-            header = [_('username'), _('email'), ]
-            data = ([u.username, u.email] for u in
-                    (User.objects.all().iterator()))
+            header = [_('username'), _('email'), _('name'), _('country') ]
+            data = (
+                [u.username, u.email, u.profile.name, u.profile.country] if hasattr(u,'profile')
```

Comment:
It will be better to use try/catch here as with that we can also log msg that profile does not exist which will help us in future debugging.